### DISCUSSION TITLE
Show full name rather than account username in GitHub

### DISCUSSION THREAD

**Discussion by frozenprakash** [community/community#19276] [Repositories]
We have around 10k+ devs in our organization and we can't ask every single one to create new duplicate git id just to user for organization.

Even GitHub suggest the same to use same id for personal and professional use to avoid duplication and to capture the stats.
https://github.community/t/using-one-account-for-all-your-projects/10197

But username will not be readable or identifiable with a person in team, so proper feature should allow user to show their full name by default in all GitHub repos and activities.

**Comment by lucalves** [comment]
Hi @frozenprakash, how are you?

If I understood the doubt, I believe this might help you:  [Managing the display of member names in your organization](https://docs.github.com/en/organizations/managing-organization-settings/managing-the-display-of-member-names-in-your-organization)

You can check? 😉

**Reply by frozenprakash** [reply]
@lucalves 
But only the company controls this information in organization.
We need this option to be available for repo admin.

**Reply by p0fi** [reply]
This will not help to display Profile Names in the new projects e.g.

**Comment by cyberfury10** [comment]
Hi, As a workaround, I have created an extension **WebEdit Pro -** ([Chrome extension](https://chrome.google.com/webstore/detail/webedit-pro/poacbppnkkaljgcfcgmabigfjhbkfdle), [Edge addon](https://microsoftedge.microsoft.com/addons/detail/webedit-pro/iglmlbhnbljiecdengccffmjlcgbjnei)) that can replace the user name with the desired full name.

Let's say usernames are random alphanumeric strings

K43234, K43235 & so on. Imagine there are 100 users under that repository with auto-generated alphanumeric IDs and whenever there is something happened in the repo. we will see

"K43234 committed a new change"
"K43235 raised a new PR"

With the above chrome extension & mapping as follows,

K43234 -> "John"
K43235 -> "Bruce"

The result would be:
"John committed a new change"
"Bruce raised a new PR"

I added capabilities to perform bulk edit (like in Postman), it is easy to add hundreds of username mapping at once

One can get the username and full name from the **_people_**  or **_Teams_** tab,

<img width="1657" alt="Screenshot 2023-08-24 at 2 10 11 PM" src="https://github.com/community/community/assets/45261873/73fcd2db-41e9-4a66-b646-128f37df744f">

here is the repo [link](https://github.com/cyberfury10/web-edit-pro) if you are interested to see the code & video [link ](https://www.youtube.com/watch?v=5j3OCeYzKFY)on how to use

**Comment by dtinth** [comment]
This is now available as a **GitHub Enterprise** feature.

See: https://github.blog/changelog/2026-02-13-you-can-now-show-profile-names-alongside-user-handles/

Example: Commit authors in `microsoft/playwright` repo shows the profile name first, then user handles. https://github.com/microsoft/playwright - The same goes for issues and pull requests. Even when tagging a user inside a comment, it displays the full name.
